src/RestrictedTokenFaucet.sol

```diff
@@ -0,0 +1,59 @@
+pragma solidity >=0.5.0;
+
+interface ERC20Like {
+    function balanceOf(address) external view returns (uint256);
```

Comment:
Both `balanceOf` and `transfer` are public in DSToken, so it might be a good idea to mirror that.

Reply:
if i change this to:
```
    function balanceOf(address) public view returns (uint256);
    function transfer(address,uint256) public; // return bool?
```
then i get an error that `Functions in interfaces must be declared external.` Am I misunderstanding your comment?

Reply:
Sorry my bad, I thought this was an abstract contract rather than an interface.